storageapi/src/vespa/storageapi/mbusprot/storagecommand.h

```diff
@@ -21,6 +21,8 @@ class StorageCommand : public mbus::Message, public StorageMessage {
     api::StorageMessage::SP getInternalMessage() override { return _cmd; }
     api::StorageMessage::CSP getInternalMessage() const override { return _cmd; }
 
+    bool hasBucketSequence() const override { return false; }
```

Comment:
If we'll be using this to signal sequencing, should it also be explicitly added to `StorageReply`? Both commands and replies need sequencing for storage API stuff

Reply:
Yes, probably. I will try and see what it is used for currently first.